README.md

```diff
@@ -11,7 +11,7 @@ To get you started you can simply clone the `java-report-generator` repository a
 
 You need [git][git] to clone the `java-report-generator` repository.
 
```

Comment:
I miss updating the JDK link at the bottom of this README.md file. I would point to Eclipse Temurin (which is the JDK version to be used by CSW projects)

Reply:
Agree. Updated.